backend/read_data_test.py

```diff
@@ -56,27 +56,28 @@ def test_unbias_all_fp_values():
     assert (unbiased_df['confusion_value'] == 'TN').all()
 
 
-#comment this in when you need to <3
-# def test_filter_by_factor():
-#     """Test the filter_by_factor function."""
-#     filtered_df = filter_by_factor(test_data, 'Gender', 'Female')
-#     assert len(filtered_df) == 3  
-#     assert (filtered_df['Gender'] == 'Female').all() 
+# comment this in when you need to <3
+def test_filter_by_factor():
+    """Test the filter_by_factor function."""
+    filtered_df = filter_by_factor(test_data, 'Female')
+    assert len(filtered_df) == 3  
```

Comment:
What does this 3 mean?

Reply:
That's one of the ones that Tara wrote, I think it verifies the length of the filtered data frame.